docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -486,6 +486,42 @@ The following command operators are supported:
   UDP
     Not implemented ("-").
 
+%RESPONSE_FLAGS_FULL%
+  Additional details about the response or connection, if any. For TCP connections, the response codes mentioned in
+  the descriptions do not apply. The values output will be a pascal case string. Possible values are:
+
+  HTTP and TCP
+    * **NoHealthyUpstream**: No healthy upstream hosts in upstream cluster in addition to 503 response code.
```

Comment:
I think it would be better to combine the docs for RESPONSE_FLAGS_FULL and RESPONSE_FLAGS to prevent drift. Maybe make the RESPONSE_FLAGS section mention both formatters, and on each line list both the short string and full string, followed by the docs for that line.